Assets/Scripts/Player/Combat/BasicAttack.cs

```diff
@@ -16,6 +16,9 @@ public class BasicAttack : MonoBehaviour
         [SerializeField] private float _force = 10f;
         [SerializeField] private Rigidbody _rb;
 
+        [Header("Player preferences")]
+        [SerializeField] bool _cursorDirection = true;
```

Comment:
Would `_freeAim` be a better name? As it opposed being locked into the direction the player is facing?